ci.py

```diff
@@ -163,6 +173,13 @@ def create_release(self):
     def create_tags(self):
         pass
 
+    def validate_config(self):
+        for dep in self.config['deps']:
+            if not os.path.abspath(dep['path']).startswith(os.getcwd()):
```

Comment:
to protect from unintended (or malicious) deletion. e.g. if `path: ../../../../sources/pmm-server` 


e